src/config.cpp

```diff
@@ -40,6 +40,7 @@ void EmulatorConfig::load() {
 
 			discordRpcEnabled = toml::find_or<toml::boolean>(general, "EnableDiscordRPC", false);
 			usePortableBuild = toml::find_or<toml::boolean>(general, "UsePortableBuild", false);
+			romsPath = toml::find_or(general, "RomsPath","");
```

Comment:
toml::find_or should take a template argument indicating the type to search for to avoid being too vague